keyboards/orthograph/info.json

```
@@ -0,0 +1,207 @@
+{
+    "keyboard_name": "Orthograph",
+    "url": "https://github.com/vladkvit",
+    "maintainer": "vladkvit",
+    "manufacturer": "vladkvit",
+    "debounce": 4,
+    "usb": {
+        "vid": "0x564B",
+        "pid": "0x0602",
+        "device_version": "1.0.0"
+    },
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
```

no elite-c pins have been defined and `readme.md` mentions Pro Micro
```suggestion
    "development_board": "promicro",
```